python/cairo-ec/src/cairo_ec/circuit_utils.cairo

```diff
@@ -6,12 +6,13 @@ func hash_full_transcript{poseidon_ptr: PoseidonBuiltin*}(limbs_ptr: felt*, n: f
     alloc_locals;
     local BASE = 2 ** 96;
 
-    let elements_end = &limbs_ptr[n * N_LIMBS];
+    local elements_end: felt* = limbs_ptr + (n * N_LIMBS);
 
     tempvar elements = limbs_ptr;
     tempvar pos_ptr = cast(poseidon_ptr, felt*);
 
     loop:
+    let N_LIMBS_HINT = N_LIMBS;
```

Comment:
This is only for the rust hint?
You cannot access N_LIMBS directly ?

Reply:
An idea why ?
Should an issue be created ?

Reply:
the way variables are accessed is a bit different, I don't think it's worth tracking as it's non-blocking and minor